fud/fud/stages/interpreter.py

```diff
@@ -0,0 +1,146 @@
+from fud.stages import Stage, SourceType, Source
+from pathlib import Path
+import simplejson as sjson
+import numpy as np
+from fud.stages.verilator.numeric_types import FixedPoint, Bitnum
+from fud.errors import InvalidNumericType
+from fud.stages.verilator.json_to_dat import parse_fp_widths
+from calyx.utils import float_to_fixed_point
+
+
+from ..utils import shell, TmpDir
+
+_FILE_NAME = "data.json"
```

Comment:
This global constant is used. Why is it prefixed with `_`?

Reply:
It's just to indicate a "private" variable not needed outside this file. It's been a while since I've written python, but I believe this is the accepted syntax?

Reply:
Got it. A comment above the constant to saying this would be useful?

Reply:
👍🏻